logstash.go

```diff
@@ -50,13 +50,14 @@ func NewHookWithFieldsAndConnAndPrefix(conn net.Conn, appName string, alwaysSent
 	return &Hook{conn: conn, appName: appName, alwaysSentFields: alwaysSentFields, hookOnlyPrefix: prefix}, nil
 }
 
-//Make a new hook which does not forward to logstash, bu simply enforces the prefix rules
-func NewFilterHook() (*Hook, error) {
+//Make a new hook which does not forward to logstash, but simply enforces the prefix rules
```

Comment:
The comment should start with the function name and then what it does.
For example:
>NewFilterHook creates a new hook which...